bin/index.js

```diff
@@ -10,9 +10,11 @@ const path = require('path');
 const config = require('../config');
 const inquirer = require('../inquirer');
 const yaml = require('js-yaml');
+const toml = require('toml');
 const TruffleConfig = require('@truffle/config');
 const { parseTrace } = require('../tracer');
 const Api = require('../api');
+const { confirmPasswordReset } = require('firebase/auth');
```

Comment:
Mmh not sure this is needed, why did you add that? It doesn't seem to be used anywhere either